components/general/typography/_typography.scss

```diff
@@ -1,100 +1,67 @@
-h1,
-h2 {
-    font-weight: 800;;
-    margin: 0 0 10px;
+.heading {
     color: $font-color;
-    line-height: 1.4;
-}
-h3,
-h4,
-h5,
-h6 {
-    font-weight: 700;
-    margin: 0 0 10px;
-    color: $font-color;
-    line-height: 1.4;
-}
+    font-family: $font-family;
+    font-weight: 400;
+    margin: 0 0 15px;
+    font-size: 14px;
+    text-transform: uppercase;
 
-h1 {
-    font-size: 24px;
+    &--bodoni-font {
```

Comment:
Yes, I agree with that, this is useful. I think to delete this classes where `font-family` exists and just left it to developers when will coding site and they will add this font which is necessary at the moment. But the question is how do you add fonts in fractal?